src/main/webapp/app/overview/course-conversations/layout/conversation-header/conversation-header.component.html

```diff
@@ -21,6 +21,13 @@ <h3 class="conversation-name d-inline-block rounded p-2 info" (click)="openConve
         <div>
             <div class="btn-toolbar p-2" role="toolbar">
                 <div class="btn-group me-2" role="group">
+                    <a
+                        *ngIf="getAsChannel(activeConversation)?.subTypeReferenceId"
+                        class="btn btn-outline-secondary sub-type-reference"
+                        [routerLink]="metisService.getLinkForChannelSubType(getAsChannel(activeConversation))"
```

Comment:
can you create helper variables for these? Using methods here calls them over and over again even though these are constants.

Reply:
ok, I use helper variables now and also replaced other places where these methods were called in the template before.